uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -90,33 +90,75 @@ public String acquireToken(@NonNull final String username,
 
 
     // click on button acquire token silent
-    public String acquireTokenSilent() throws UiObjectNotFoundException {
+    public String acquireTokenSilent() throws UiObjectNotFoundException, InterruptedException {
         final UiObject acquireTokenSilentButton = UiAutomatorUtils.obtainUiObjectWithResourceId("com.msft.identity.client.sample.local:id/btn_acquiretokensilent");
         scrollToElement(acquireTokenSilentButton);
+        Thread.sleep(2000);
         acquireTokenSilentButton.click();
         final UiObject result = UiAutomatorUtils.obtainUiObjectWithResourceId("com.msft.identity.client.sample.local:id/txt_result");
         return result.getText();
     }
 
     // click on button getUsers
-    public List<String> getUsers() throws UiObjectNotFoundException {
-        UiAutomatorUtils.handleButtonClick("com.msft.identity.client.sample.local:id/btn_getUsers");
-
+    public List<String> getUsers() throws UiObjectNotFoundException, InterruptedException {
+        final UiObject getUsersButton = UiAutomatorUtils.obtainUiObjectWithResourceId("com.msft.identity.client.sample.local:id/btn_getUsers");
+        scrollToElement(getUsersButton);
+        Thread.sleep(2000);
```

Comment:
why is this sleep added?

Reply:
to make sure before clicking UI is updated already

Reply:
I mean, even after scrolling to the element getUsersButton, would you need this delay?